source/tutorials/importing.rst

```diff
@@ -76,6 +113,36 @@ Importing data
      --source-format CasavaOneEightSingleLanePerSampleDirFmt \
      --output-path demux.qza
 
+Casava 1.8 paired-end demultiplexed fastq
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format description
+******************
+
+In this format, there are two ``fastq.gz`` file for each sample in the study, and the file name includes the sample identifier. The forward and reverse read file names for a single sample might look like ``L2S357_15_L001_R1_001.fastq.gz`` and ``L2S357_15_L001_R2_001.fastq.gz``, respectively. The underscore-separated fields in this file name are the sample identifier, the barcode sequence or a barcode identifier, the lane number, the read number, and the set number.
+
+Obtaining example data
+**********************
+
+.. download::
+  :url: https://data.qiime2.org/2017.02/tutorials/importing-sequence-data/casava-18-paired-end-demultiplexed.zip
+  :saveas: casava-18-paired-end-demultiplexed.zip
+
+.. command-block::
+
+  unzip -q casava-18-paired-end-demultiplexed.zip
+
+Importing data
+**************
+
+.. command-block::
+  qiime tools import \
+    --type 'SampleData[PairedEndSequencesWithQuality]' \
+    --input-path casava-18-paired-end-demultiplexed \
+    --source-format CasavaOneEightSingleLanePerSampleDirFmt \
+    --output-path demux.qza
```

Comment:
This command will fail when run with `make html` because the output name has already been used (overwriting outputs is disallowed by `command-block`).